ui/app/models/metrics/config.js

```diff
@@ -1,10 +1,32 @@
 import DS from 'ember-data';
+import { computed } from '@ember/object';
+// import { apiPath } from 'vault/macros/lazy-capabilities';
```

Comment:
Should these be removed?

Reply:
Yes, but this is a good reminder to make another follow-up ticket for handling non-operator policies